PowerPointLabs/Test/UnitTest/HighlightLab/BaseHighlightLabTest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+using Test.Util;
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace Test.UnitTest.HighlightLab
+{
+    [TestClass]
+    public class BaseHighlightLabTest : BaseUnitTest
```

Comment:
Sure it's possible. But bcos the rest of highlight lab features are implemented in a legacy way, they're neither SOLID nor testable. So unless someone migrates it, it can only be tested in FT